### ISSUE TITLE
Improve like link on mobile

### ISSUE THREAD

**Issue by buddh4** [humhub/humhub#4919]
The like link consists of two links one for liking/unliking and one for opening a list of people who liked this post. On mobile (and even on desktop) its hard to click on the small counter in order to openen the list of people who liked the content. Also its possible to accidentally click on the like link instead of the counter.

There should be a better seperation between like link and counter.

**Comment by ArchBlood** [comment]
What about a hold function? Meaning when someone presses down and holds the button it then opens the liked list?

**Comment by buddh4** [comment]
Hmm, this behavior would also not be really obvious. But on mobile such hold gestures are more commen, so maybe would be a possible solution. But the user would need to figure this out by himself, which is not optimal.

**Comment by ArchBlood** [comment]
> Hmm, this behavior would also not be really obvious. But on mobile such hold gestures are more commen, so maybe would be a possible solution. But the user would need to figure this out by himself, which is not optimal.

Good point, the hold method would also probably make users think that reactions similar to that of Facebook and etc. were possible :thinking:
